### PR TITLE
[INTERNAL] v2: Change security audit reporting level to high or critical only

### PR DESCRIPTION
**Pull request by d3xter666**
People should be motivated to migrate to v3. 
We shall still have some awareness regarding the vulnerabilities that might appear in v2 with a higher severity.

### CODE REVIEW

File: audit-ci.jsonc [hunk 1]
```diff
@@ -1,32 +1,6 @@
...
 	// $schema provides code completion hints to IDEs.
 	"$schema": "https://github.com/IBM/audit-ci/raw/main/docs/schema.json",
-	"low": true,
-	"allowlist": [
-
-		// The package "taffydb" is only used during jsdoc build and contains only data for jsdoc generation.
-		// After jsdoc build finished, db is destroyed.
-		// According to JSDoc, this should not be considered a security risk:
-		// https://github.com/jsdoc/jsdoc/tree/main/packages/jsdoc-salty#taffydb-jsdoc-and-security
-		"GHSA-mxhp-79qh-mcx6",
```
Comment (matz3) [inline audit-ci.jsonc:11]: Isn't this one "high"?

File: audit-ci.jsonc [hunk 2]
```diff
@@ -1,32 +1,6 @@
...
-
-		// The package "http-cache-semantics" is vulnerable to a Regular Expression Denial of Service (ReDoS) attack.
-		// It is used by the "make-fetch-happen" and "got" packages which are only used to communicate with the npm registry configured by the user (registry.npmjs.org by default).
-		// Although this ReDoS attack is mainly applicable to servers, in theory a server could also send malicious headers to the client (UI5 Tooling) to cause an unexpected slowdown.
-		// However, this configured npm registry is already considered a trusted connection as code is downloaded and run by the client.
-		"GHSA-rc47-6667-2j5j",
```
Comment (matz3) [inline audit-ci.jsonc:17]: Also high according to GitHub.

File: audit-ci.jsonc [hunk 3]
```diff
@@ -1,32 +1,6 @@
...
-
-		// "cacheable-request" has a dependency to "http-cache-semantics" (GHSA-rc47-6667-2j5j) which is 
-		// why it is considered as high severity. Not applicable as described above for GHSA-rc47-6667-2j5j.
-		"GHSA-8x6c-cv3v-vp6g",
```
Comment (matz3) [inline audit-ci.jsonc:21]: Also high but has been withdrawn see: https://github.com/advisories/GHSA-8x6c-cv3v-vp6g

File: audit-ci.jsonc (hunk 1 above)
Comment (d3xter666) [reply audit-ci.jsonc]: Thanks! I ran it locally on with v3 dependencies and it made me think wrongly about it as lower vulnerability. Now should be ok

**Review by matz3 — APPROVED** [review]

**Pull request merged**